Automattic/wp-calypso
# Untangling: apply new design to Settings -> Administration -> Delete site

Fixes https://github.com/Automattic/dotcom-forge/issues/9862

## Proposed Changes

This PR applies the new global view design to Settings -> Administration -> Delete site flow.

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Open /sites.
2. Click a site.
3. Go to Settings -> Administration.
4. Click the `Delete your site permanently` button.
5. Verify you see this:

<img width="997" alt="image" src="https://github.com/user-attachments/assets/a1829939-7f1b-442c-b83b-5b13c1984db9">

6. Submit the form, verify the site is deleted.
7. REGRESSION: Transfer another site via `/settings/general/:slug`; verify the flow can still be completed.

## Pre-merge Checklist
